notebooks/nightly_report/nightly_report_ts_version.ipynb

```diff
@@ -1,4 +1,4 @@
~{
```

Comment:
<p>seeing the screenshot, please  make  sure that top-left and bottom-left panels are aligned . Same for top-right and bottom-right panels.  Adding line that connects the dots would make all plots here more readable. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>